src/main/java/gg/projecteden/nexus/models/scheduledjobs/ScheduledJobs.java

```diff
@@ -0,0 +1,71 @@
+package gg.projecteden.nexus.models.scheduledjobs;
+
+import dev.morphia.annotations.Converters;
+import dev.morphia.annotations.Entity;
+import dev.morphia.annotations.Id;
+import gg.projecteden.mongodb.serializers.LocalDateTimeConverter;
+import gg.projecteden.mongodb.serializers.UUIDConverter;
+import gg.projecteden.nexus.framework.persistence.serializer.mongodb.JobConverter;
+import gg.projecteden.nexus.models.PlayerOwnedObject;
+import gg.projecteden.nexus.models.scheduledjobs.common.AbstractJob;
+import gg.projecteden.nexus.models.scheduledjobs.common.AbstractJob.JobStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Data
+@Builder
+@Entity(value = "scheduled_jobs", noClassnameStored = true)
+@NoArgsConstructor
+@AllArgsConstructor
+@RequiredArgsConstructor
+@Converters({UUIDConverter.class, JobConverter.class, LocalDateTimeConverter.class})
+public class ScheduledJobs implements PlayerOwnedObject {
+	@Id
+	@NonNull
+	private UUID uuid;
+	private Map<JobStatus, Set<AbstractJob>> jobs = new HashMap<>();
```

Comment:
worried this object may grow too large for the database, but given that the only time this has happened was with cyn's transaction history and that we don't have an alternative system rn, it's probably fine

Reply:
https://github.com/ProjectEdenGG/Nexus/pull/20/files/2a10fafbd9fec602746bccfc3d45255b586c2e63#diff-ace30cdf0a5982ea89aec7db8032e23221c439cbde970fdbd7435add0581e346R66-R69

I will manually keep an eye on the errored/interrupted jobs, and will probably add them to the janitor at some point when i feel more confident about the system